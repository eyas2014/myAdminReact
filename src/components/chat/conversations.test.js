import React from 'react';
import Conversations from './Conversations';
import enzyme from 'enzyme';

var messages = [
	{
		displayName:"Mike Doe",
		displayPicture: "dist/img/user4-128x128.jpg",
		time: "2:15",
		content: "I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market",
		attachment: {
			groupName: "Attachments",
			fileName: "Theme-thumbnail-image.jpg"
		}
	},
	{
		displayName: "Alexander Pierce",
		displayPicture: "dist/img/user3-128x128.jpg",
		time: "5:15",
		content: "I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market"
	},
	{
		displayName: "Susan Doe",
		displayPicture: "dist/img/user2-160x160.jpg",
		time: "5:30",
		content: "I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market"
	}
];



const wrapper=enzyme.shallow(<Conversations data={messages}></Conversations>);

it ('should match the snapshot', ()=>{
	expect(wrapper).toMatchSnapshot();
})
