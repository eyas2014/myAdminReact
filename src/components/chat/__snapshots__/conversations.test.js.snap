// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Conversations
    data={
      Array [
        Object {
          "attachment": Object {
            "fileName": "Theme-thumbnail-image.jpg",
            "groupName": "Attachments",
          },
          "content": "I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market",
          "displayName": "Mike Doe",
          "displayPicture": "dist/img/user4-128x128.jpg",
          "time": "2:15",
        },
        Object {
          "content": "I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market",
          "displayName": "Alexander Pierce",
          "displayPicture": "dist/img/user3-128x128.jpg",
          "time": "5:15",
        },
        Object {
          "content": "I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market",
          "displayName": "Susan Doe",
          "displayPicture": "dist/img/user2-160x160.jpg",
          "time": "5:30",
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <ChatMessage
          attachment={
            Object {
              "fileName": "Theme-thumbnail-image.jpg",
              "groupName": "Attachments",
            }
          }
          content="I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market"
          displayName="Mike Doe"
          displayPicture="dist/img/user4-128x128.jpg"
          time="2:15"
        />,
        <ChatMessage
          content="I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market"
          displayName="Alexander Pierce"
          displayPicture="dist/img/user3-128x128.jpg"
          time="5:15"
        />,
        <ChatMessage
          content="I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market"
          displayName="Susan Doe"
          displayPicture="dist/img/user2-160x160.jpg"
          time="5:30"
        />,
      ],
      "id": "chat-box",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "0",
        "nodeType": "class",
        "props": Object {
          "attachment": Object {
            "fileName": "Theme-thumbnail-image.jpg",
            "groupName": "Attachments",
          },
          "content": "I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market",
          "displayName": "Mike Doe",
          "displayPicture": "dist/img/user4-128x128.jpg",
          "time": "2:15",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "1",
        "nodeType": "class",
        "props": Object {
          "attachment": undefined,
          "content": "I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market",
          "displayName": "Alexander Pierce",
          "displayPicture": "dist/img/user3-128x128.jpg",
          "time": "5:15",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "2",
        "nodeType": "class",
        "props": Object {
          "attachment": undefined,
          "content": "I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market",
          "displayName": "Susan Doe",
          "displayPicture": "dist/img/user2-160x160.jpg",
          "time": "5:30",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <ChatMessage
            attachment={
              Object {
                "fileName": "Theme-thumbnail-image.jpg",
                "groupName": "Attachments",
              }
            }
            content="I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market"
            displayName="Mike Doe"
            displayPicture="dist/img/user4-128x128.jpg"
            time="2:15"
          />,
          <ChatMessage
            content="I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market"
            displayName="Alexander Pierce"
            displayPicture="dist/img/user3-128x128.jpg"
            time="5:15"
          />,
          <ChatMessage
            content="I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market"
            displayName="Susan Doe"
            displayPicture="dist/img/user2-160x160.jpg"
            time="5:30"
          />,
        ],
        "id": "chat-box",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "class",
          "props": Object {
            "attachment": Object {
              "fileName": "Theme-thumbnail-image.jpg",
              "groupName": "Attachments",
            },
            "content": "I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market",
            "displayName": "Mike Doe",
            "displayPicture": "dist/img/user4-128x128.jpg",
            "time": "2:15",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "1",
          "nodeType": "class",
          "props": Object {
            "attachment": undefined,
            "content": "I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market",
            "displayName": "Alexander Pierce",
            "displayPicture": "dist/img/user3-128x128.jpg",
            "time": "5:15",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "2",
          "nodeType": "class",
          "props": Object {
            "attachment": undefined,
            "content": "I would like to meet you to discuss the latest news about the arrival of the new theme. They say it is going to be one the best themes on the market",
            "displayName": "Susan Doe",
            "displayPicture": "dist/img/user2-160x160.jpg",
            "time": "5:30",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
